configure.py

```diff
@@ -1,599 +1,381 @@
+  parser.add_argument('--build-v8-with-gn',
+                      dest='_ignored',
+                      help=argparse.SUPPRESS)
+  # End backwards compatibility
+  parser.add_argument('--code-cache-path',
+                      help='Use a file generated by tools/generate_code_cache.js to compile the'
+                           ' code cache for builtin modules into the binary')
+  parser.add_argument('--without-ssl',
+                      action='store_true',
+                      help='build without SSL (disables crypto, https, inspector, etc.)')
+  parser.add_argument('--without-node-options',
+                      action='store_true',
+                      help='build without NODE_OPTIONS support')
+  parser.add_argument('--ninja',
+                      action='store_true',
+                      dest='use_ninja',
+                      help='generate build files for use with Ninja')
+  parser.add_argument('--enable-asan',
+                      action='store_true',
+                      help='build with asan')
+  parser.add_argument('--enable-static',
+                      action='store_true',
+                      help='build as static library')
+  parser.add_argument('--no-browser-globals',
+                      action='store_true',
+                      help='do not export browser globals like setTimeout, console, etc. ' +
+                           '(This mode is not officially supported for regular applications)')
+  parser.add_argument('--without-inspector',
+                      action='store_true',
+                      help='disable the V8 inspector protocol')
+  parser.add_argument('--shared',
+                      action='store_true',
+                      help='compile shared library for embedding node in another project. ' +
+                           '(This mode is not officially supported for regular applications)')
+  parser.add_argument('--without-v8-platform',
+                      action='store_true',
+                      default=False,
+                      help='do not initialize v8 platform during node.js startup. ' +
+                           '(This mode is not officially supported for regular applications)')
+  parser.add_argument('--without-bundled-v8',
+                      action='store_true',
+                      default=False,
+                      help='do not use V8 includes from the bundled deps folder. ' +
+                           '(This mode is not officially supported for regular applications)')
+  parser.add_argument('--verbose',
+                      action='store_true',
+                      default=False,
+                      help='get more output from this script')
+  parser.add_argument('--v8-non-optimized-debug',
+                      action='store_true',
+                      default=False,
+                      help='compile V8 with minimal optimizations and with runtime checks')
+  parser.add_argument('extra_gyp_args',
```

Comment:
This is not necessary for Python 3 compatibility I believe. Can you explain why this is introduced?